pixel/theme.json

```diff
@@ -369,32 +371,34 @@
 			},
             "h1": {
                 "typography": {
-                    "fontSize": "3rem"
+                    "fontSize": "clamp(1.375rem, 1.2916666666666667rem + 0.4166666666666667vw, 1.625rem)"
                 }
             },
             "h2": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--x-large)"
+                    "fontSize": "clamp(1.25rem, 1.2083333333333333rem + 0.20833333333333334vw, 1.375rem)"
                 }
             },
             "h3": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--large)"
+                    "fontSize": "clamp(1.125rem, 1.0833333333333333rem + 0.20833333333333334vw, 1.25rem)"
                 }
             },
             "h4": {
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--medium)"
+                    "fontSize": "1.125rem"
                 }
             },
             "h5": {
                 "typography": {
-                    "fontSize": "1.125rem"
+                    "fontSize": "0.813rem",
```

Comment:
What is the source for sizes and clamp ranges?

In the Figma file, for instance, `h5` is set to `14px`, so in theory this should be `1rem` if we don't change the current root font size.

Reply:
The root font size is `16px`, this should be standard and unchanged unless I'm missing something. 

Sorry, should have specified — the clamp range is based on a 320px - 1280px range.

Reply:
[Gutenberg](https://github.com/WordPress/gutenberg/blob/trunk/lib/block-supports/typography.php#L372) is always assuming min as `48rem` and max as `100rem` for the fluid typography. I guess it is best to use the same range.